public/index.html

```diff
@@ -10,50 +10,52 @@
-                    class="s-input s-input__md grid--cell" />
-                <button type="submit" class="s-btn s-btn__outlined s-btn__muted s-btn__md bg-white ws-nowrap grid--cell">Show browsers</button>
+                    class="s-input s-input__md grid--cell bc-orange-3" />
+                <button type="submit" class="s-btn s-btn__outlined s-btn__muted s-btn__md fc-white bg-orange-800 baw0 ws-nowrap grid--cell">Show browsers</button>
             </div>
         </div>
     </form>
 
-    <div id="js-error-container" class="mx-auto w100 wmx12 px64 sm:pl24 sm:pr24 wmx12 w100 d-none">
+    <div id="js-error-container" class="grid--cell mx-auto w100 wmx10 px64 sm:pl24 sm:pr24 d-none">
         <aside class="s-notice s-notice__danger s-notice__important js-error-message" role="alert" aria-hidden="false">
             Error
         </aside>
     </div>
 
-    <div id="js-data-container" class="mx-auto w100 wmx12 px64 sm:pl24 sm:pr24 wmx12 w100">
-        <div class="ta-center fs-title">
+    <div id="js-data-container" class="grid fd-column mx-auto w100 wmx10 px64 sm:pl32 sm:pr32">
+        <div class="ta-center fs-title mb32">
             Global coverage <strong id="js-global-coverage">0.00%</strong>
         </div>
-    
-        <div class="grid gs48 mt0 fw-wrap">
-            <div class="grid--cell fl-equal">
-                <div class="fs-title fw-bold mb24 ta-center">Mobile browsers</div>
-                <div id="js-mobile-container"></div>
-            </div>
-            <div class="grid--cell fl-equal">
+
+        <div class="grid gs64 jc-center md:ai-center md:fd-column">
+            <div class="grid--cell fl-equal w100 wmx4">
                 <div class="fs-title fw-bold mb24 ta-center">Desktop browsers</div>
                 <div id="js-desktop-container"></div>
             </div>
+            <div class="grid--cell fl-equal w100 wmx4">
+                <div class="fs-title fw-bold mb24 ta-center">Mobile browsers</div>
+                <div id="js-mobile-container"></div>
+            </div>
         </div>
+    </div>
 
-        <div class="grid gs32 py32 fw-wrap">
-            <div class="grid--cell">Designed for use by <a href="https://stackoverflow.design">Stacks</a></div>
-            <div class="grid--cell">Source hosted at <a href="https://github.com/b-kelly/browserslist-viewer">Github</a></div>
-            <div class="grid--cell">Logos by <a href="https://github.com/alrra/browser-logos">alrra/browser-logos</a></div>
-            <div class="grid--cell">Data by <a href="https://github.com/ben-eb/caniuse-lite">caniuse-lite</a> <strong class="js-caniuse-version"></strong></div>
-            <div class="grid--cell">Functionality by <a href="https://github.com/browserslist/browserslist">browserslist</a> <strong class="js-browserslist-version"></strong></div>
+    <div class="grid--cell mt-auto mx-auto w100 wmx10 px64 sm:pl32 sm:pr32">
+        <div class="pt64 pb24 fc-light s-anchors s-anchors__grayscale">
```

Comment:
Those dang `gs` classes always troll me when it comes to horizontal scrolling. Flex gutters can not come soon enough.